### ISSUE TITLE
dependency not found 

### ISSUE THREAD

**Issue by har1s1m** [ulivz/vue-foldable#4]
Hi
I was trying to use this package(yarn add vue-foldable -s) but I couldn't

To solve dependency not found issue, I started to learn more about npm and webpack)

Firstly, in the package.json, main/module/unpkg are in "dist" folder, however, there is not "dist" folder, so I made necessary changes, and now I got other error - "Failed to resolve loader: stylus-loader" 

Help me plz

**Comment by ulivz** [comment]
Could you provide a reproduction repo? Thanks.

**Comment by har1s1m** [comment]
yarn add vue-foldable -s

gave me the following structure: 

```
-vue-foldable
  |_ src
	|_ index.js
	|_ VueFoldable.vue
	|_ VueIcon.vue
  |_ CHANGELOG.md
  |_ package.json
  |_ README.md
```

and in package.json:

```
{
  "name": "vue-foldable",
  "version": "1.0.0-alpha.0",
  "description": "foldable component for vue.js",
  "repository": {
    "url": "git+https://github.com/ulivz/vue-foldable.git",
    "type": "git"
  },
  "main": "dist/vue-foldable.cjs.js",
  "module": "dist/vue-foldable.es.js",
  "unpkg": "dist/vue-foldable.js",
  "jsdelivr": "dist/vue-foldable.js",
  "sideEffects": false,
  "bugs": {
    "url": "https://github.com/ulivz/vue-foldable/issues"
  },
  "homepage": "https://github.com/ulivz/vue-foldable#readme",
  "author": "ULIVZ <chl814@foxmail.com>",
  "license": "MIT",
  "keywords": [
    "vue",
    "foldable",
    "callapse",
    "summary"
  ]
}


```

**Comment by zyrianov-v-fd** [comment]
Still relevant, need to use your plugin, but can't.

**Comment by ulivz** [comment]
Hey, sorry for the delay of response, I just updated the dist tag, you can re-install and try again.

![image](https://user-images.githubusercontent.com/23133919/51177513-bb9c8b00-18fa-11e9-966b-e4d58f7596d2.png)

**Comment by idealatom** [comment]
+1 for still having this issue. There is only "src" folder in your package, so this import 
```
import VueFoldable from 'vue-foldable'
```
raises error: Module not found: Error: Can't resolve 'vue-foldable'

**Comment by superbiche** [comment]
+1, same as @idealatom

**Comment by ulivz** [comment]
@idealatom @superbiche Sorry for the delay, I just published `1.0.0` and it contains `dist` directory.

https://unpkg.com/vue-foldable@1.0.0/
